spikeinterface/widgets/amplitudes.py

```diff
@@ -17,15 +19,25 @@ class AmplitudesWidget(BaseWidget):
         List of unit ids.
     segment_index: int
         The segment index (or None if mono-segment)
+    max_spikes_per_unit: int
+        Number of max spikes per unit to display. Use None for all spikes.
+        Default 500.
     hide_unit_selector : bool
-        For sortingview backend, if True the unit selector is not displayed
+        If True the unit selector is not displayed
+        (sortingview backend)
+    plot_histogram : bool
+        If True, an histogram of the amplitudes is plotted on the right axis 
+        (matplotlib backend)
+    bins : int
+        If plot_histogram is True, the number of bins for the amplitude histogram.
+        If None (default), this is automatically adjusted.
     """
     possible_backends = {}
 
     
     def __init__(self, waveform_extractor: WaveformExtractor, unit_ids=None, unit_colors=None,
-                 segment_index=None, hide_unit_selector=False, plot_histograms=False,
-                 bins=None, backend=None, **backend_kwargs):
+                 segment_index=None, max_spikes_per_unit=500, hide_unit_selector=False, 
+                 plot_histograms=False, bins=None, backend=None, **backend_kwargs):
```

Comment:
I think for amplitudes drift checking having all is still good.
Maybe we could have num_spike_per_seconds.